src/module.c

```diff
@@ -438,13 +445,54 @@ int TSDB_range(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
         return REDISMODULE_ERR;
     }
 
-    ReplySeriesRange(ctx, series, start_ts, end_ts, aggObject, time_delta);
+    size_t cmdLen;
+    Direction direction = Forward;
+    RedisModule_StringPtrLen(argv[0], &cmdLen);
+    if (cmdLen == strlen("ts.revrange")) {
```

Comment:
performance wise we prefer to register two function instead of doing strcompare between the two. I did that mistake with INCR/DECR and Id prefer not to do that again.

Reply:
I need to know my direction for different functions. It is an enum.